commit 7a48c82c11fd8f38c46f351de30122cf46581f77
Author: George Amanakis <gamanakis@gmail.com>
Date:   Thu Jun 13 00:03:38 2019 -0400

    mvebu: fixes commit a7e68927d0
    
    err_free_stats has been deprecated. Replace with err_netdev.
    
    Compile-tested on: mvebu
    Runtime-tested on: mvebu
    
    Signed-off-by: George Amanakis <gamanakis@gmail.com>

diff --git a/target/linux/mvebu/patches-4.14/403-net-mvneta-convert-to-phylink.patch b/target/linux/mvebu/patches-4.14/403-net-mvneta-convert-to-phylink.patch
index 577317a84d..34a2d342db 100644
--- a/target/linux/mvebu/patches-4.14/403-net-mvneta-convert-to-phylink.patch
+++ b/target/linux/mvebu/patches-4.14/403-net-mvneta-convert-to-phylink.patch
@@ -905,7 +905,7 @@ Signed-off-by: Russell King <rmk+kernel@arm.linux.org.uk>
 +	phylink = phylink_create(dev, dn, phy_mode, &mvneta_phylink_ops);
 +	if (IS_ERR(phylink)) {
 +		err = PTR_ERR(phylink);
-+		goto err_free_stats;
++		goto err_netdev;
 +	}
 +
 +	pp->phylink = phylink;
